clj-refactor.el

```diff
@@ -538,13 +538,18 @@ _s_: Refactor related functions
 
 (defun cljr--search-forward-within-sexp (s &optional save-excursion)
   "Searches forward for S in the current sexp.
-
-if SAVE-EXCURSION is T POINT does not move."
-  (let ((bound (save-excursion (forward-list 1) (point))))
-    (if save-excursion
-        (save-excursion
-          (search-forward s bound t))
-      (search-forward s bound t))))
+   S must be followed by a space, ), or ] character.
```

Comment:
Why? Is it meant to detect the end of a symbol? If so `\\_>` is more appropriate.